Source/HYPFormsCollectionViewDataSource.m

```diff
@@ -78,11 +79,7 @@ - (instancetype)initWithCollectionView:(UICollectionView *)collectionView
 
 - (NSMutableArray *)collapsedForms
 {
-    if (_collapsedForms) return _collapsedForms;
-
-    _collapsedForms = [NSMutableArray new];
-
-    return _collapsedForms;
+    return _collapsedForms = (_collapsedForms) ?: [NSMutableArray new];
```

Comment:
Or maybe I'm being an idiot here, I'm gonna revert it just to be sure that it does not mess things up big-time :)